src/Photos/Enums.cs

```diff
@@ -246,8 +272,7 @@ public enum PHAuthorizationStatus : long {
 	[Mac (10,12, onlyOn64 : true)]
 	[TV (10,0)]
 	[Native]
-	public enum PHAssetResourceType : long
-	{
+	public enum PHAssetResourceType : long {
```

Comment:
your style is better - but don't rewrite existing code (unless required)
i.e. minimize _damage_ (changes) in your PRs to ease reviewers life :)

Reply:
Got it sorry about that!

Reply:
I will undo some of the other style changes I made